frontend/packages/local-storage-operator-plugin/src/utils/index.ts

```diff
@@ -15,23 +15,19 @@ export const getNodes = (
   selectedNodes: string[],
 ): string[] => (showNodes ? selectedNodes : allNodes);
 
-export const getLabelIndex = (
-  nodeSelector: NodeAffinityTerm[],
-  label: string,
-  operator: string,
+export const getNodeSelectorTermsIndices = (
+  nodeSelectorTerms: { matchExpressions: MatchExpression[]; matchFields: MatchExpression[] }[],
```

Comment:
```suggestion
  nodeSelectorTerms: { matchExpressions: MatchExpression[]; matchFields: MatchExpression[] }[] = [],
```
So that we can avoid lodash.